controllers/vspherecluster_reconciler.go

```diff
@@ -158,12 +154,6 @@ func (r *clusterReconciler) reconcileDelete(ctx context.Context, clusterCtx *cap
 		log := log.WithValues("VSphereMachine", klog.KObj(vsphereMachine))
 		ctx := ctrl.LoggerInto(ctx, log)
 
```

Comment:
Missed this before. Shouldn't we be able to drop the entire finalizer handling here?

Reply:
xref: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/pull/1284/files

cc @chrischdi

Reply:
You're right - we can get rid of the entire block that deals with deleting Machines here.